docs/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 ### Changed
 
+* PlanContext::graph() renamed to PlanContext::graph_mut()
```

Comment:
Missing the link to this pull request. See previous entries for an example. (also don't forget to include the link, and not just the pr number).

Reply:
Ah I missed that thanks. It should be okay now